lib/document.js

```diff
@@ -2646,8 +2649,12 @@ Document.prototype.validateSync = function(pathsToValidate, options) {
     shouldValidateModifiedOnly = this.$__schema.options.validateModifiedOnly;
   }
 
+  let pathsToSkip = get(options, 'pathsToSkip');
```

Comment:
Let's just do `pathsToSkip = options.pathsToSkip`, `get()` is handy for dotted paths and cases where we don't know whether `options` exists, but it also has some non-trivial performance overhead.